apps/emqx_dashboard/src/emqx_dashboard_swagger.erl

```diff
@@ -417,13 +417,17 @@ init_prop(Keys, Init, Type) ->
         fun(Key, Acc) ->
             case hocon_schema:field_schema(Type, Key) of
                 undefined -> Acc;
-                Schema -> Acc#{Key => to_bin(Schema)}
+                Schema -> Acc#{Key => format_prop(Key, Schema)}
```

Comment:
Q: what is the meaning of the `prop`

Reply:
It is inherited from `init_prop`, probably means "properties".